erpnext/controllers/stock_controller.py

```diff
@@ -581,6 +581,8 @@ def get_gl_entries(self, warehouse_account=None, default_expense_account=None, d
 						).format(wh, self.company)
 					)
 
+		gl_list = make_regional_gl_entries(gl_list, self)
```

Comment:
What kind of changes are you looking for in your custom app (for stock)? Which doctype is it?

`get_gl_entries` is internally used by `make_gl_entries` and overriding one should be enough in my opinion.
So if you wish to update stock gl entries for sales invoice, you can do so with regional override created above.

Reply:
I was thinking about special use case with Materiel Renting  where stock valuation need to be adapted.
But, Scopen only had a prospect now with this feature, let's remove it the form this PR, to keep only on Sales Invoices regional feature., I've updated the PR title

Reply:
[This](https://github.com/frappe/erpnext/blob/develop/erpnext/controllers/buying_controller.py#L336) hook could be useful for all purchases.

Can be done similarly for Sales if needed.